README.md

```diff
@@ -37,22 +44,133 @@ api("com.github.spotify:openfeature-kotlin-sdk:[ANY_BRANCH]-SNAPSHOT")
 
 This will get a build from the head of the mentioned branch. 
 
-### Resolving a flag
+## 🌟 Features
+
+- support for various backend [providers](https://openfeature.dev/docs/reference/concepts/provider)
+- easy integration and extension via [hooks](https://openfeature.dev/docs/reference/concepts/hooks)
+- bool, string, numeric, and object flag types
+- [context-aware](https://openfeature.dev/docs/reference/concepts/evaluation-context) evaluation
+
+## 🚀 Usage
```

Comment:
I wonder if we should link to some "static-context paradigm" high-level description here (e.g. https://github.com/open-feature/spec/blob/main/specification/glossary.md#static-context-paradigm)

Reply:
I suppose OF would need to change it to also reflect mobile clients first? Otherwise might be a bit confusing.

Reply:
Ok let's skip this then, we can add relevant links later if needed